src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```
@@ -575,6 +581,8 @@ The .NET Foundation licenses this file to you under the MIT license.
       <!-- see https://github.com/emscripten-core/emscripten/issues/16836 for the issue that means we have to force the linker to include these symbols before LTO -->
       <!--<CustomLinkerArg Condition="'$(Optimize)' != 'true'" Include="$(WasmOptimizationSetting) -flto" /> -->
       <CustomLinkerArg Condition="'$(IlcLlvmTarget)' != ''" Include="-target $(IlcLlvmTarget)" />
+
+	    <CustomLinkerArg Condition="'$(WasmExternalDwarf)' == 'true'" Include="-gseparate-dwarf" />
```

```suggestion
	    <CustomLinkerArg Condition="'$(WasmExternalDwarf)' == 'true'" Include="-gseparate-dwarf" />
```
I think we should pass something like `GetFilenameWithoutExtension($(NativeBinary)).debug.wasm` here to avoid the odd `.wasm.debug.wasm` thing.